libraries/cyclestreets-view/src/main/res/values/strings.xml

```diff
@@ -33,8 +33,11 @@
   <string name="route_fetching_existing">Fetching route from CycleStreets.net, please wait</string>
   <string name="route_finding_failed">Route finding failed</string>
   <string name="route_loading">Loading route, please wait</string>
+  <string name="route_not_found" translatable="false">Route not found</string>
```

Comment:
Any reason for making these new strings not translatable?  If it was just "I'm still developing" then fine, but we should change this when all other comments are addressed, and add placeholders (with the English values) in the different languages XML so they're ready to be translated.

Reply:
Being lazy, avoiding errors in the build for now.
I'll add them in.
btw Is there an easy way for a translator to see which strings still need translating, other than reading through them all?

Reply:
Dunno, Google it and see 🙂

Reply:
(Still todo: add in all the translation placeholders - will wait til end in case of any changed wording suggestions.)